dts/msm8909/apq8009w-wtp.dts

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+	qcom,msm-id = <265 0>, <301 0>;
+	qcom,board-id= <8 0x105>;
+
+	model = "Qualcomm Technologies, Inc. APQ8009W-PM8916 MTP";
+	compatible = "qcom,apq8009-mtp", "qcom,apq8009", "qcom,mtp";
```

Comment:
I am confused. `MTP` or `WTP`?
https://android.googlesource.com/kernel/msm/+/android-msm-mullet-3.18-oreo-wear-dr/arch/arm/boot/dts/qcom/apq8009w-wtp.dts#24
https://android.googlesource.com/kernel/msm/+/android-msm-mullet-3.18-oreo-wear-dr/arch/arm/boot/dts/qcom/apq8009w-1gb-wtp.dts#24

Reply:
WTP is a thing, but I think you wouldn't want to do this as they use custom board IDs.

Reply:
`qcom,board-id= <8 0x105>;` fits `apq8009w-wtp.dts`. I would assume it's shared by several devices.
For board IDs, do you have downstream dts for some references?

Reply:
I see https://github.com/Nexusma/msm-catfish-9.0-wear-os/blob/master/arch/arm/boot/dts/qcom/catfish.dts#L20-L21

However, `CDP` is supposed to be `<8 0x103>`
https://android.googlesource.com/kernel/msm/+/android-msm-mullet-3.18-oreo-wear-dr/arch/arm/boot/dts/qcom/apq8009w-cdp.dts#27